### PR TITLE
Add default year to the roadmap

### PR DESCRIPTION
**Pull request by SenghoungLim**
## Description
Create a new function
Compare if the length of the data array is 0
If so, add the new Default
        (Thanks Ryan for Helping me out)
        
## Screenshots
image

## Final Checks:
- [ ] Verify successful deployment
- [ ] Delete branch

(optional)
- [ ] Write tests
- [ ] Write documentation

### CODE REVIEW

File: site/src/pages/RoadmapPage/Planner.tsx [hunk 1]
```diff
@@ -235,25 +235,39 @@ const Planner: FC = () => {
     }
   }
 
+  const initializePlanner = () => {
+    if (data.length == 0) {
```
Comment (ethanwong16) [inline site/src/pages/RoadmapPage/Planner.tsx:239]: Would be good to add a comment here that explains there is a future TODO once we have support for multiple planners: to ensure that a default new year is added only when a new planner is created, not when an existing planner is empty.

**Review by ethanwong16 — APPROVED** [review]
Great job @SenghoungLim ! LGTM 🥳 

I left a suggestion to add a comment explaining how a new year would always be created whenever the planner is empty and how we might want to refactor this later, but other than that feel free to merge and squash the branch when you're ready!

**Comment by ethanwong16** [comment]
Closes #194

File: site/src/pages/RoadmapPage/Planner.tsx (hunk 1 above)
Comment (SenghoungLim) [reply site/src/pages/RoadmapPage/Planner.tsx:239]: Will do in the code, thank you!

**Pull request merged**